### PR TITLE
Abstract out the version number in our getting started doc so as to a…

### PR DESCRIPTION
**Pull request by geoffjentry**
…void confusion

### CODE REVIEW

File: docs/tutorials/FiveMinuteIntro.md [hunk 1]
```diff
@@ -20,15 +20,15 @@ At the end of this five minute introduction you will have:
...
 
-* Head down to the bottom of the page and find the link to download `cromwell-29.jar`. Note that when you read these instructions, the latest Cromwell version might be higher than 29. That's fine, these instructions will work just as well! Download the jar file.
- 	 * WARNING! If you're on a Mac, the security settings might try to stop you from running Cromwell! Don't worry, if this happens just go to `System Preferences > Security & Privacy > General` and find the `cromwell` jar listed on the page. Click `Open anyway`. The `cromwell-29.jar` will now automatically download to your `Downloads` directory.
+* Look for the latest version at the top of the page, and find the link to download the jar. It'll have a name like `cromwell-XYZ.jar` where `XYZ` is the version. Download the jar file.
+* WARNING! If you're on a Mac, the security settings might try to stop you from running Cromwell! Don't worry, if this happens just go to `System Preferences > Security & Privacy > General` and find the `cromwell` jar listed on the page. Click `Open anyway`. The `cromwell-XYZ.jar` will now automatically download to your `Downloads` directory.
```
Comment (cjllanwarne) [inline docs/tutorials/FiveMinuteIntro.md]: Maybe just `cromwell-xy.jar`? At least until we reach version 100?

Comment (geoffjentry) [reply docs/tutorials/FiveMinuteIntro.md]: @cjllanwarne Your ability to out-pedantic me never ceases to am[aze|use] me

Comment (cjllanwarne) [reply docs/tutorials/FiveMinuteIntro.md]: I'll chose to take that as a complement

**Comment by mcovarr** [comment]
Blocked on BCS slowness "addressed" by this PR https://github.com/broadinstitute/cromwell/pull/5102

**Comment by mcovarr** [comment]
rebased and ready to merge

**Pull request merged**